README.md

```diff
@@ -131,11 +162,13 @@ _this is way too simplistic. needs an update_
 - Kibana Plugin
 - Hosted Data Portal generating service
 
-### Development
+---
+
+### Development Details
 
 Arranger is a [lerna](https://github.com/lerna/lerna) flavored [monorepo](https://medium.com/@maoberlehner/monorepos-in-the-wild-33c6eb246cb9). The modules exposed by Arranger compose all of the necessary code required to build an application such as the [Genomic Data Commons](https://portal.gdc.cancer.gov/).\*
 
-### Releases
+#### Releasing Instructions
 
 - From `master` branch, run `npm run tag <version>`
 - Publishing process will be run [by Jenkins](https://jenkins.qa.cancercollaboratory.org/blue/organizations/jenkins/Overture.bio%2Farranger/activity?branch=master)
```

Comment:
Is this applicable to the user?  It points to our internal Jenkins.  What would the user have to do themselves to publish?  Have their own Jenkins pipeline?  Is this necessary for them?

Reply:
This is something the docs already had, to indicate to developers how to release a new version.

While making all these changes, I was thinking this better belongs in a standalone contribution guide, but it's TODO for the future.